backend/generator.js

```diff
@@ -18,8 +18,12 @@ exports.executeScript = function (socket, formData) {
   var uniqueId = uuidV4();
   var webUI = "true";
   var subProject = "";
+  var subDocpath = [];
   if (formData.subProject !== undefined) {
-    subProject = formData.subProject.join(",")
+    subProject = formData.subProject.join(",");
+    for(i=0; i<formData.subProject.length; i++){
+        subDocpath.push('docs');
+    }
```

Comment:
@imujjwal96 @sch00lb0y I realized that if `SUBPROJECT_URLS` is defined but `SUBPROJECT_DOCPATHS` is not, would lead to problems, hence as a temporary fix, from the webUI defining `SUBPROJECT_DOCAPTHS`. Please review this change.